is-array/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for is-array 1.0
+// Project: https://github.com/retrofox/is-array
+// Definitions by: Pine Mizune <https://github.com/pine>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = isArray;
+declare function isArray(val?: any): boolean;
```

Comment:
Please make the argument required.

Thanks for taking the time to set up the typings :).